src/lib.rs

```diff
@@ -1,6 +1,27 @@
+use elements::{BlockHash, Txid};
+
+pub mod avg_vbytes;
+pub mod client;
+mod coin_selection;
+mod estimate_transaction_size;
 pub mod input;
 pub mod loan;
 pub mod oracle;
 pub mod swap;
+pub mod wallet;
```

Comment:
A re-export of `Wallet` might be worthwhile so we can say `baru::Wallet` in waves instead of `baru::wallet::Wallet`.

Reply:
Where possible, we should strive for a flat public API. Most of the time, this means having almost all modules inside private and be explicit about our public API through re-exports.

This also helps with stabilizing the public API because you can re-arrange things internally without making breaking changes.